README.md

```diff
@@ -20,3 +20,25 @@ For other issues, please use the following repos:
 - For class library and framework functioning issues, file on [dotnet/corefx](https://github.com/dotnet/corefx) repo
 - For runtime issues, file on [dotnet/coreclr](https://github.com/dotnet/coreclr) issues
 
+Build Status
+------------
+
+|Ubuntu 14.04 |Ubuntu 16.04 |Debian 8.2 |Windows x64 |Windows x86 |Mac OS X |CentOS 7.1 |RHEL 7.2 |Fedora 23 |OpenSUSE 13.2 |
```

Comment:
The build information is definitely not "keep in sync across repos".  These build statuses can only be displayed here in the core-setup repo.

The same goes for the current version below.  There is no place in the CLI readme that had that information.

I trimmed the "Installer" links from here because of the exact thing you mention here - that info is already contained in the cli readme, so I didn't feel it was correct to duplicate it here.